src/lib/components/FormModel/form-model.test.js

```diff
@@ -1,17 +1,12 @@
 import React from 'react';
-import { render, cleanup } from 'react-testing-library';
+import { render, cleanup } from '@testing-library/react';
 import FormModel from './form-model.component';
 import 'jest-dom/extend-expect';
 
 afterAll(cleanup);
 
-const setup = () => <FormModel model={{}} formValues={{}} />;
-
-describe('FormModel Component', () => {
-  const component = setup();
-  const { container } = render(component);
-
-  it('should renders without crashing', () => {
-    expect(container).toBeTruthy();
-  });
-});
+/**
+ * Form model wires all the functionality together. Logic for the individual
+ * components should be in their respective tests
+ */
+test('FormModel Component', () => {});
```

Comment:
I'm not sure what this does. The comment specifies well that individual components are tested in their own scope, so this may just be a `jest` thing that I'm not familiar with.

Reply:
This one may be able to be deleted. I spoke with @angel-araya-jd about it, and he couldn't think of a valid test for this control.